js/common/modules/@arangodb/foxx/check-args.js

```diff
@@ -32,6 +33,7 @@ const mimeTypes = require('mime-types');
 const mediaTyper = require('media-typer');
 const ArangoError = require('@arangodb').ArangoError;
 const ERROR_BAD_PARAMETER = require('@arangodb').errors.ERROR_BAD_PARAMETER;
+const ajv = new Ajv();
```

Comment:
AJV is a JSON Schema validator that is fairly fast and lightweight. We don't expose it to users directly, they just need to pass JSON schemas as JS objects and we derive a validation function from that.